unit/analyses/call_graph.cpp

```diff
@@ -14,34 +14,10 @@ Module: Unit test for call graph generation
 #include <analyses/call_graph_helpers.h>
 
 #include <util/symbol_table.h>
-#include <util/std_code.h>
 
 #include <goto-programs/goto_convert_functions.h>
 
-static symbolt create_void_function_symbol(
-  const irep_idt &name,
-  const codet &code)
-{
-  const code_typet void_function_type({}, empty_typet());
-  symbolt function;
-  function.name=name;
-  function.type=void_function_type;
-  function.mode=ID_java;
-  function.value=code;
-  return function;
-}
-
-static bool multimap_key_matches(
-  const std::multimap<irep_idt, irep_idt> &map,
-  const irep_idt &key,
-  const std::set<irep_idt> &values)
-{
-  auto matching_values=map.equal_range(key);
-  std::set<irep_idt> matching_set;
-  for(auto it=matching_values.first; it!=matching_values.second; ++it)
-    matching_set.insert(it->second);
-  return matching_set==values;
-}
+#include "call_graph_test_utils.h"
```

Comment:
There is `unit/testing-utils/` and I'm wondering whether that would be a better place to put it? Along with that I'd ask for a cpp file to be added that will take the definitions while the header file would just have the declarations.